cmake_common_scripts/cmake/cmake_tools.cmake

```diff
@@ -304,7 +307,10 @@ endmacro()
 
 # This extracts package name and version to the following variables
 # ${prefix}_extracted_name and ${prefix}_extracted_version
-# This was taken from https://github.com/ros-industrial/abb_robot_driver, author Jon Tjerngren
+# This was taken from https://github.com/ros-industrial/abb_robot_driver
+# Author: Jon Tjerngren
+# Copyright (c) 2020, ABB Schweiz AG
+# License: BSD 3-Clause
```

Comment:
I've seen projects separate out code under different licenses by storing things in separate files.

That would also make it easy to attach the appropriate license: you could just include the license header in the file which falls under that license.

You could then of course still `include(..)` that file in this one.

Reply:
That works. Should I remove all LICENSE files from the repository?